apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTestsPostgreSql.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data.CommonTesting;
+using Google.Cloud.Spanner.V1;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests;
+
+[Collection(nameof(SpannerDatabaseFixturePostgreSql))]
+[CommonTestDiagnostics]
+[Trait(Constants.SupportedOnEmulator, Constants.No)]
+public class BindingTestsPostgreSql
```

Comment:
I'm guessing we have an equivalent test class for Google dialect. Could we have a `BindingTestsBase` class that receives a base fixture and implements all the common tests, and then two dialect specific classes with the traits and with the tests for the dialect specific types? Just so that we don't have all the duplicated code.